llvm/test/CodeGen/SystemZ/builtin-setjmp.ll

```diff
@@ -0,0 +1,123 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
```

Comment:
Here I think it would be good to add one more test, to verify that the *frame pointer* is saved if it is live - this is currently tested nowhere.  To do so, I think it should suffice to call setjmp in a function that required a frame pointer because it does a variable-sized alloca.  (This FP test could be done both with and without mbackchain to test all the permuations.)